tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py

```diff
@@ -16,140 +16,397 @@
           preparsing_validators=[
               validators.hasLength(150),
           ],
-          postparsing_validators=[],
+          postparsing_validators=[
+            validators.validate__FAM_AFF__SSN(),
```

Comment:
nit: the docstring for this validator should probably be updated to reference the labels instead of the item numbers since we re-use for multiple record layouts.